### ISSUE TITLE
Recent changes in ERA5 netcdf files break the topo_scale functionality

### ISSUE THREAD

**Issue by MatteaE** [ArcticSnow/TopoPyScale#114]
Since ~August 2024, CDS is delivering ERA5 files in a new form which is not backwards-compatible: among other things, dimension "date" has been renamed to "valid_date". Full list of changes is [here](https://confluence.ecmwf.int/display/CKB/GRIB+to+netCDF+conversion+on+new+CDS+and+ADS+systems#GRIBtonetCDFconversiononnewCDSandADSsystems-ERA5).

According to [this post](https://forum.ecmwf.int/t/new-time-format-in-era5-netcdf-files/3796/8), the old format is (as of December 2024) accessible by passing "data_format": "netcdf_legacy" within the dictionary which is supplied to cdsapi.Client().retrieve(), but this functionality may be discontinued in the future.

Thus there may be some updates needed within topo_scale.py to support the new format, starting at line ~168 - (...).sel(time=tvec.values).

**Comment by joelfiddes** [comment]
exactly.....there have been other things as well.....My approach has been to convert the new format to the old format in a small converter. I have started to add support for grib which is more stable.

**Comment by joelfiddes** [comment]
see https://github.com/ArcticSnow/TopoPyScale/issues/110#issue-2521740135

**Comment by joelfiddes** [comment]
fixes in branch CDSbeta merged to main - closing this.
